src/test/addresses_mainnet.sol

```diff
@@ -21,460 +21,464 @@ contract Addresses {
+        addr["MCD_JOIN_RETH_A"]                  = 0xC6424e862f1462281B0a5FAc078e4b63006bDEBF;
+        addr["MCD_CLIP_RETH_A"]                  = 0x27CA5E525ea473eD52Ea9423CD08cCc081d96a98;
+        addr["MCD_CLIP_CALC_RETH_A"]             = 0xc59B62AFC96cf9737F717B5e5815070C0f154396;
+        addr["DIRECT_HUB"]                       = 0x12F36cdEA3A28C35aC8C6Cc71D9265c17C74A27F;
+        addr["DIRECT_MOM"]                       = 0x1AB3145E281c01a1597c8c62F9f060E8e3E02fAB;
+        addr["DIRECT_COMPV2_DAI_POOL"]           = 0x621fE4Fde2617ea8FFadE08D0FF5A862aD287EC2;
+        addr["DIRECT_COMPV2_DAI_PLAN"]           = 0xD0eA20f9f9e64A3582d569c8745DaCD746274AEe;
+        addr["DIRECT_COMPV2_DAI_ORACLE"]         = 0x0e2bf18273c953B54FE0a9dEC5429E67851D9468;
+        addr["DIRECT_AAVEV2_DAI_POOL"]           = 0x66aE0574Eb28B92c82569b293B856BB99f80F040;
+        addr["DIRECT_AAVEV2_DAI_PLAN"]           = 0x5846Aee09298f8F3aB5D837d540232d19e5d5813;
+        addr["DIRECT_AAVEV2_DAI_ORACLE"]         = 0x634051fbA31829E245C616e79E289f89c8B851c2;
+        addr["DIRECT_SPARK_DAI_POOL"]            = 0xAfA2DD8a0594B2B24B59de405Da9338C4Ce23437;
+        addr["DIRECT_SPARK_DAI_PLAN"]            = 0x104FaDbb7e17db1A685bBa61007DfB015206a4D2;
+        addr["DIRECT_SPARK_DAI_ORACLE"]          = 0xCBD53B683722F82Dc82EBa7916065532980d4833;
+        addr["GNO"]                              = 0x6810e776880C02933D47DB1b9fc05908e5386b96;
+        addr["PIP_GNO"]                          = 0xd800ca44fFABecd159c7889c3bf64a217361AEc8;
+        addr["MCD_JOIN_GNO_A"]                   = 0x7bD3f01e24E0f0838788bC8f573CEA43A80CaBB5;
+        addr["MCD_CLIP_GNO_A"]                   = 0xd9e758bd239e5d568f44D0A748633f6a8d52CBbb;
+        addr["MCD_CLIP_CALC_GNO_A"]              = 0x17b6D0e4237ea7F880aF5F58257cd232a04171D9;
+        addr["SPARK_PROXY"]                      = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
+        addr["CRON_SEQUENCER"]                   = 0x238b4E35dAed6100C6162fAE4510261f88996EC9;
+        addr["CRON_AUTOLINE_JOB"]                = 0x67AD4000e73579B9725eE3A149F85C4Af0A61361;
+        addr["CRON_LERP_JOB"]                    = 0x8F8f2FC1F0380B9Ff4fE5c3142d0811aC89E32fB;
+        addr["CRON_D3M_JOB"]                     = 0x1Bb799509b0B039345f910dfFb71eEfAc7022323;
+        addr["CRON_CLIPPER_MOM_JOB"]             = 0xc3A76B34CFBdA7A3a5215629a0B937CBDEC7C71a;
+        addr["CRON_ORACLE_JOB"]                  = 0xe717Ec34b2707fc8c226b34be5eae8482d06ED03;
+        addr["CRON_FLAP_JOB"]                    = 0xc32506E9bB590971671b649d9B8e18CB6260559F;
+        addr["MCD_PSM_GUSD_A_JAR"]               = 0xf2E7a5B83525c3017383dEEd19Bb05Fe34a62C27;
+        addr["MCD_PSM_GUSD_A_INPUT_CONDUIT_JAR"] = 0x6934218d8B3E9ffCABEE8cd80F4c1C4167Afa638;
+        addr["MCD_PSM_PAX_A_JAR"]                = 0x8bF8b5C58bb57Ee9C97D0FEA773eeE042B10a787;
+        addr["MCD_PSM_PAX_A_INPUT_CONDUIT_JAR"]  = 0xDa276Ab5F1505965e0B6cD1B6da2A18CcBB29515;
```

Comment:
Shouldn't these be closer to the other PSM contracts?
Even though they are not technically related, they are still kind of part of the same structure.

Reply:
done